shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/model/Answer.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.avishek.phong.kmmic.domain.model
+
+interface Answerable {
+    val id: String
+    val content: String?
+    val placeholder: String?
+}
+
+data class Answer(
+    override val id: String,
+    val text: String?,
+    val displayOrder: Int,
+    var inputMaskPlaceholder: String?
```

Comment:
How about we rename `inputMaskPlaceholder` to `placeholder` instead since this model is in principle independent of the API model?

Reply:
solved in efe216b